veniq/dataset_collection/dataflow/preprocess.py

```diff
@@ -0,0 +1,36 @@
+import re
+from pathlib import Path
+
+from veniq.utils.encoding_detector import read_text_with_autodetected_encoding
+
+
+def create_existing_dir(directory: Path):
+    if not directory.exists():
+        directory.mkdir(parents=True)
+
+
+def preprocess(file: str):
```

Comment:
return type?

Reply:
each node returns a dict

Reply:
so please specify that, including the contents of the dict

Reply:
it's ugly, I've done this. 'Cause it can have any value -> `Dict[str, Any]` doesn't tell you smth important

Reply:
that s why i asked to come up with custom types